Runtime/Native/INativeClient.cs

```diff
@@ -40,5 +40,11 @@ internal interface INativeClient
         /// Disable native integration
         /// </summary>
         void Disable();
+
+        /// <summary>
+        /// Pause ANR thread

```

Comment:
Is it worth exposing "Thread" here? Seems like an implementation detail.

Reply:
I'm against it - I don't see a use case of it and can only generate more issue that we think. Right now we're managing it and to do that right, user need to have an access to nativeClient implementation which is internal right now.